packages/client/src/Config.ts

```diff
@@ -109,7 +109,7 @@ export interface ControlLayerConfig {
      * The server is used by the network layer to accept incoming connections
      * over the public internet to improve the network node's connectivity.
      */
-    websocketPortRange?: PortRange
+    websocketPortRange?: PortRange | null
```

Comment:
Should we update the comment above as follows?

> If not specified, a server will not be started.

into

> If *set to null*, a server will not be started.

Reply:
Updated